docs/website/blog/2023-02-01-odo-dotnet-postgresql-openshift.mdx

```diff
@@ -0,0 +1,523 @@
+
+You can see that our application is now using the in-memory database. We will start the PostgreSQL database and connect our application in the following steps.
+
+You can leave `odo dev` running in the terminal and open a new terminal window to continue with the next steps. Make sure that you run all `odo` commands in the the root folder  of our .NET application (where you have `devfile.yaml` and `RazorPagesContacts` files).
+
+## Use the PostgreSQL database
+
+### Start a new PostgreSQL instance on the OpenShift cluster
+
+To start a new PostgreSQL instance on the OpenShift cluster, we can use the `postgresql-ephemeral` template that is available in the OpenShift cluster.
+
+If you have `oc` CLI installed, you can use the following command to start a new PostgreSQL instance:
+
+```sh
+oc new-app postgresql-ephemeral
+````
+<details>
+  <summary>example output</summary>
+  <div>
+
+  ```
+  ▶ oc new-app postgresql-ephemeral
+  --> Deploying template "openshift/postgresql-ephemeral" to project tkral-dev
+
+      PostgreSQL (Ephemeral)
+      ---------
+      PostgreSQL database service, without persistent storage. For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/.
+
+      WARNING: Any data stored will be lost upon pod destruction. Only use this template for testing
+
+      The following service(s) have been created in your project: postgresql.
+
+              Username: user2GR
+              Password: 6UrRpvC8ups70ffr
+        Database Name: sampledb
+        Connection URL: postgresql://postgresql:5432/
+
+      For more information about using this template, including OpenShift considerations, see https://github.com/sclorg/postgresql-container/.
+
+      * With parameters:
+          * Memory Limit=512Mi
+          * Namespace=openshift
+          * Database Service Name=postgresql
+          * PostgreSQL Connection Username=user2GR # generated
+          * PostgreSQL Connection Password=6UrRpvC8ups70ffr # generated
+          * PostgreSQL Database Name=sampledb
+          * Version of PostgreSQL Image=10-el8
+
+  --> Creating resources ...
+      secret "postgresql" created
+      service "postgresql" created
+      deploymentconfig.apps.openshift.io "postgresql" created
+  --> Success
+      Application is not exposed. You can expose services to the outside world by executing one or more of the commands below:
+      'oc expose service/postgresql'
+      Run 'oc status' to view your app.
+  ```
+
+  </div>
+</details>
+
+If you don't have `oc` CLI installed, you can use the OpenShift web console to start a new PostgreSQL instance.
+
+<details>
+  <summary>Starting new PostgreSQL instance using OpenShift Web Console</summary>
+  <div>
+  <ol>
+    <li>
+      Log in to  OpenShift Web Console.
+    </li>
+    <li>
+      Switch to the **Developer** view and make sure that you are using correct project.
```

Comment:
It seems the bold for **Developer** is not taken into account here

Reply:
fixed. MDX is weird, if you have text inside <Tab> you have to use HTML tags :-(